corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -285,7 +284,7 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
                       on: $root.forceRequiredVisible,
                     }">{% trans 'Sorry, this response is required!' %}</div>
   <!-- /ko -->
-  <!-- ko if: datatype() === 'info' -->
+  <!-- ko if: isLabel -->
```

Comment:
This comment is nonblocking and not a result of this PR so I don't expect any changes but I think it's confusing how I think this instance of "isLabel" is checking if the entry is an instance of `InfoEntry` [here](https://github.com/dimagi/commcare-hq/blob/1cefe1ec6d1f3fc8dd65cd4af9bd35b5c7f90cff/corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js#L1319). But in `getEntry`, there's other checks that's named `isLabel` which checks the style, and not related to `datatype()==='info'`. I found that confusing, and i think it would be nice to clear up. 

This instance is fine since it's pulling from  the [Question] attribute (https://github.com/dimagi/commcare-hq/blob/946e797a1dc88d44cff2112da5189fdeae9b2004/corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js#L768)

Reply:
Good callout, thank you. I'll look into the two different definitions and see if it makes sense to rename one of them here for clarity.

Reply:
⭐